ambry-api/src/main/java/com/github/ambry/config/ServerConfig.java

```diff
@@ -127,7 +127,18 @@ public class ServerConfig {
   @Config("server.repair.requests.db.factory")
   public final String serverRepairRequestsDbFactory;
 
+  /**
+   * Server execution mode
+   * - Data recovery mode
+   * - Data verification mode
+   * - Data serving mode
+   */
+  public static final String SERVER_EXECUTION_MODE = "server.execution.mode";
+  @Config("server.execution.mode")
+  public final String serverExecutionMode;
```

Comment:
There is a method in VerifiableProperty to getEnum from string. We can use Enum type here directly